assets/js/script.js

```diff
@@ -0,0 +1,150 @@
+$(function () {
+  var $box = $('.box')
+  var $locBtn = $('#loc')
+  var $colorBtn = $('#colorBtn')
+  var $startBtn = $('#startBtn')
+  var $stopBtn = $('#stopBtn')
+  var $resetBtn = $('#resetBtn')
+  var textbox = $('#textbox')
+  var colorArr = ['red', 'blue', 'orange', 'yellow', 'white', 'green', 'grey', 'purple']
+  var recArr = []
+  var recCol = []
+  var count = 0
+  var index = 0
+  var on = 0
+  var refreshIntervalId
+  var n = 2 // Difficulty Setting
+  var $turn = $('#turn')
+  var scoreObj = {
+    score: 0,
+    scoreBoard: $('#scoreBoard')
+  }
+  var gameOverDiv = $('.gameOver')
+  var $restartBtn = $('.restartBtn')
+
+  $resetBtn.on('click', reset)
+  $stopBtn.on('click', stop)
+  $locBtn.on('click', compareResult)
+  $(document).on('keydown', function (e) {
+    if (e.keyCode === 37) {
+      compareResult()
+    }
+  })
+  $restartBtn.on('click', () => {
+    gameOverDiv.css('display', 'none')
+  })
+  $colorBtn.on('click', compareColor)
+  $(document).on('keydown', function (e) {
```

Comment:
could you give some spaces in between these codes? group them logically so it's easier to be read